The following arguments were used to generate this output.
[-technique, selection, -coverage, function, -order, absolute, -origOrder, ambari-new/ambari-server/target/ambari-server-auto-order, -testInputDir, ambari/ambari-server/target/sootTestOutput-auto-selection, -filesToDelete, ambari-new/ambari-server/target/ambari-server-env-files, -project, Ambari-Server, -testType, auto, -oldVersCFG, ambari/ambari-server/target/selectionOutput, -newVersCFG, ambari-new/ambari-server/target/selectionOutput, -getCoverage, -outputDir, /homes/gws/winglam/winglam/ambari/dependent-tests-impact/experiments/selection-results, -timesToRun, 1, -dependentTestFile, /homes/gws/winglam/winglam/ambari/dependent-tests-impact/experiments/selection-dt-list/selection-Crystal-auto-function-absolute.txt]

Execution time of TLG and its time to find/nullify any DTs for 1 machine (does not include the time to run the tests in the new order): 0.622
Number of tests selected out of total in original order: 19 / 3337
Number of DTs not fixed: 0
Number of DTs fixed: 0
APFD value: 0.3067709273180989
Execution time for executing the following testing order: 0.367

Test order list:
[RegressionTest2.test020, RegressionTest6.test147, RegressionTest0.test090, RegressionTest0.test331, RegressionTest0.test429, RegressionTest1.test004, RegressionTest1.test147, RegressionTest1.test334, RegressionTest2.test176, RegressionTest2.test236, RegressionTest2.test311, RegressionTest2.test338, RegressionTest2.test388, RegressionTest2.test448, RegressionTest2.test462, RegressionTest3.test068, RegressionTest3.test090, RegressionTest4.test302, RegressionTest6.test101]

Time each test takes to run in the new order:
[186870797, 8641699, 17185152, 10106866, 7696983, 17257206, 7985814, 20655392, 13597811, 5116495, 7550040, 8533335, 5768281, 5830526, 5296477, 20533747, 5232588, 11642161, 1187523]

Dependent test list:
[Test: randoop.crystal.RandoopTest4.test153, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test135], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test222, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test213], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test495, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test492], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test163, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test161], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test228, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test227], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test186, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test134], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test227, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test186], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test331, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test323], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test132, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test128], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test129, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test114], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test355, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test336], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test330, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test256], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test146, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test133], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test280, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test269], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test445, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test429], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test487, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test459], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test243, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test212], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test85, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test80], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test93, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test86], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test459, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test445], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test273, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test253], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test492, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test487], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test194, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test180], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test13, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test493], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test82, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test63], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test93, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test92], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest6.test63, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test130], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test130, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test171], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest6.test122, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest6.test63], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test211, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test160], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test445, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test441], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test234, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test222], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test253, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test239], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test466, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test443], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test432, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test425], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest6.test160, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest6.test154], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test121, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test102], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test488, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test364], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test128, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test111], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test128, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test100], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test55, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test47], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test97, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test53], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test100, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test93], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test166, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test158], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test466, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test460], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test47, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test30], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test199, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test194], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test18, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test221], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test114, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test109], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test111, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test100], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test422, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test420], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test86, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test85], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test100, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test55], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test22, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test10], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test271, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test234], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test192, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test122], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test239, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test211], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test317, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test273], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test447, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test445], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test476, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test447], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test493, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test477], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test311, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test280], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test212, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test163], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test443, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test432], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest6.test78, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test497], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test19, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test296], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test410, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test400], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test258, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test231], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test327, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test258], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test379, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test327], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test460, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test447], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test289, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test271], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test30, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test22], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test464, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test289], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test365, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test337], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test7, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test3], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test313, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test311], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test395, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test381], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test422, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test409], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test125, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test121], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test395, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test355], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest6.test154, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest6.test124], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test441, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test365], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest6.test124, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest6.test49], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test161, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test153], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test63, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test49], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest6.test55, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest6.test46], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest6.test159, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest6.test56], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest6.test56, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest6.test55], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test236, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test19], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test456, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test236], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test206, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test109], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test140, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test139], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test80, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test27], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test191, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test319], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test302, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test191], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test313, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test302], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test323, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test313], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test122, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test118], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test425, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test395], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test80, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test78], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test78, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test494], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test102, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test85], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test256, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test243], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test231, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test211], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test53, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test43], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test49, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test13], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test59, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test49], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test337, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test317], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test133, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test59], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test324, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test320], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test409, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test395], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test429, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test422], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test46, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test495], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test160, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test129], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test451, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test440], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test187, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test198], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test440, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test187], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test477, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test476], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test336, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test330], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test10, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test499], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test3, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test464], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test381, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test324], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test231, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test229], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test229, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test146], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test158, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test132], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test206, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test193], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test213, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test206], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test193, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test166], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test118, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test97], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test13, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test7], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test262, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test231], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test264, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test262], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test314, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test313], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test320, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test314], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test109, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test104], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test497, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test412], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest6.test46, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test362], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test362, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test336], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test211, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test192], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test92, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test75], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test180, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test175], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test175, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test128], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test492, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test490], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test484, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test473], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test473, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test466], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test490, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test484], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test400, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test382], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test44, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test373], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest6.test147, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test414], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test414, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test281], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test420, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test410], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test43, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test13], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test499, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test466], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest6.test49, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest6.test30], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test104, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test101], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test135, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test125], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test447, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test422], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test412, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test206], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test109, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test456], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test494, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test492], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test364, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test199], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest6.test30, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test488], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test269, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test264], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test75, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test48], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test49, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test46], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test382, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test379], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test101, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test93], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test85, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test82], The revealed different behavior: PASS, when executed after: []]

Coverage test list:
[34, 17, 5, 11, 5, 5, 0, 5, 5, 5, 5, 5, 0, 5, 28, 0, 0, 5, 0]
--------------------------
Total time (of all machines and iterations plus initial TestListGenerator): 1.880